<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-2GUQO5V" failures="5" tests="17" name="Test" time="73.555" errors="0" timestamp="2024-07-25T11:20:56 IST">
  <testcase classname="testCases.TC_001_AccountRegistrationTest" name="registrationTest" time="4.982"/>
  <testcase classname="testCases.TC_002_NavigateToRegister" name="testNavigateToRegisterViaRegister" time="0.797"/>
  <testcase classname="testCases.TC_002_NavigateToRegister" name="testNavigateToRegisterViaLogin_NewCustomer" time="3.734"/>
  <testcase classname="testCases.TC_002_NavigateToRegister" name="testNavigateToRegisterViaLogin_Register" time="3.702"/>
  <testcase classname="testCases.TC_003_ValidLogin" name="testValidLogin" time="1.001"/>
  <testcase classname="testCases.TC_004_LoginDataDrivenTests" name="validateLoginWithDiffCredentials" time="6.055">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateLoginWithDiffCredentials -->
  <testcase classname="testCases.TC_004_LoginDataDrivenTests" name="validateLoginWithDiffCredentials" time="1.514">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateLoginWithDiffCredentials -->
  <testcase classname="testCases.TC_004_LoginDataDrivenTests" name="validateLoginWithDiffCredentials" time="1.413">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_004_LoginDataDrivenTests.validateLoginWithDiffCredentials(TC_004_LoginDataDrivenTests.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateLoginWithDiffCredentials -->
  <testcase classname="testCases.TC_004_LoginDataDrivenTests" name="validateLoginWithDiffCredentials" time="5.911"/>
  <testcase classname="testCases.TC_004_LoginDataDrivenTests" name="validateLoginWithDiffCredentials" time="1.574"/>
  <testcase classname="testCases.TC_004_LoginDataDrivenTests" name="validateLoginWithDiffCredentials" time="5.877"/>
  <testcase classname="testCases.TC_004_LoginDataDrivenTests" name="validateLoginWithDiffCredentials" time="1.455"/>
  <testcase classname="testCases.TC_005_RegisterDataDrivenTest" name="validateRegistrationWithDiffCredentials" time="1.326">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials(TC_005_RegisterDataDrivenTest.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateRegistrationWithDiffCredentials -->
  <testcase classname="testCases.TC_005_RegisterDataDrivenTest" name="validateRegistrationWithDiffCredentials" time="1.014"/>
  <testcase classname="testCases.TC_005_RegisterDataDrivenTest" name="validateRegistrationWithDiffCredentials" time="1.011"/>
  <testcase classname="testCases.TC_005_RegisterDataDrivenTest" name="validateRegistrationWithDiffCredentials" time="0.992"/>
  <testcase classname="testCases.TC_005_RegisterDataDrivenTest" name="validateRegistrationWithDiffCredentials" time="1.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_005_RegisterDataDrivenTest.validateRegistrationWithDiffCredentials(TC_005_RegisterDataDrivenTest.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- validateRegistrationWithDiffCredentials -->
</testsuite> <!-- Test -->
